docs/components/connectors/out-of-the-box-connectors/twilio.md

```diff
@@ -23,7 +23,7 @@ The **Twilio Connector** allows you to integrate your BPMN service with Twilio's
 Before you can use the Twilio Connector, create a Twilio account and obtain an account SID and auth token from the [Twilio Console](https://www.twilio.com/console). You will also need to have a phone number to use as the sender for your SMS messages.
 
 :::note
-It is highly recommended to use Camunda secrets to store your account SID and auth token, so you don't expose sensitive information directly from the process. See [managing secrets](https://docs.camunda.org/manual/latest/user-guide/process-engine/secrets/) to learn more.
+We recommend using Camunda secrets to store your account SID and auth token so you don't expose sensitive information directly from the process. See [managing secrets](https://docs.camunda.org/manual/latest/user-guide/process-engine/secrets/) to learn more.
```

Comment:
Aaaaaand another one.

Reply:
Similar comment here:

```suggestion
Use Camunda secrets to store your account SID and auth token so you don't expose sensitive information directly from the process. See [managing secrets](https://docs.camunda.org/manual/latest/user-guide/process-engine/secrets/) to learn more.
```

Avoids "recommend" or "recommended" and is the implied recommendation.

Reply:
Agreed 👍 I'll need to create a separate issue to ensure we link to the secrets content consistently, rather than several different links.